### DISCUSSION TITLE
Is possible to know which supported wallet the user have connected to?

### DISCUSSION THREAD

**Discussion by Robsonsjre** [WalletConnect/walletconnect-monorepo#2444] [Developer Support] [answered]
In our app, we ask permission for tracking analytics. One way to track wallet integration co-branding performance would be to know what is our most used wallets. Is that possible? I only receive the "account" information after the connection. I am using web3modal 2.2.0

**Comment by finessevanes** [comment, marked as answer]
The `session.peerMeta.name` field can give information about the wallet, such as its name or type.
